Sources/ConcentricOnboarding/ConcentricOnboardingView.swift

```diff
@@ -130,9 +132,10 @@ public struct ConcentricOnboardingView : View {
                 }) { shape }
 
                 if !isAnimating.value {
-                    Image("arrow")
+                     Image(systemName: nextIcon)
```

Comment:
Extra space. As I understand this means user can only use system images now, is this right? Still better than requiring specific "arrow" name, but maybe add an option to pass the image itself? Not necessary, just a thought

Reply:
Yeah.... the user will be able to use the system image... Yeah, I also thought of allowing the user to pass the image as an option... I will update it to accept images as an option and I will prettify the code...


Thanks

Reply:
I have fixed the indentation.. but I tried to pass Image as an option.... But I think it won't work because the next icon image background is a State variable... changing the bgColor won't work if the whole image is set from the user....

but if there is a work around please let me know.